lib/vmdb/config.rb

```diff
@@ -53,7 +53,7 @@ def set_worker_setting!(klass, setting, value)
     end
 
     def save(resource = MiqServer.my_server)
-      resource.set_config(config)
+      resource.add_settings_for_resource(config)
```

Comment:
Do we also need this line here? https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_server/configuration_management.rb#L14 or maybe in `#add_settings_for_resource`? @Fryguy I'm not sure when we changed formats.

Reply:
Also, maybe just add a `resource.ntp_reload_queue if resource.respond_to(:ntp_reload_queue)`?

Both server and zone do, but region doesn't.

@Fryguy what do you think?

Reply:
It (`ntp_reload_queue` call) could also just move from `MiqServer#set_config` to `MiqServer#reload_settings` ... not sure what would be best.